src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -93,8 +95,15 @@ private void InitializeColors()
             CodeAnalysisColors.AccentBarColorKey = EnvironmentColors.FileTabInactiveDocumentBorderEdgeBrushKey;
         }
 
-        protected override void LoadComponentsInUIContext(CancellationToken cancellationToken)
+        protected override async Task LoadComponentsInUIContextAsync(CancellationToken cancellationToken)
```

Comment:
📝 Let me know what you think of `IPreloadService` in sharwell/roslyn@ded603f29bbdd8f67aed6920cce9e0e527ddc529 as a way to avoid hard-coding the services here.

Reply:
@sharwell It looks great! When can I use it?